From: SeongJae Park <sj@kernel.org>
Date: Wed, 15 Oct 2025 17:27:27 -0700
Subject: [PATCH] ==== misc cleanup ====

mm/damon: misc cleanups

Yet another batch of misc cleanups and refactoring for DAMON code,
tests, and documents.

First two patches rename DAMOS core filters related code for
readasbility.

Three following patches refactor page table walk callback functions in
DAMON, as suggested by Hugh and David, and I promised.

Next one patch refactor the drgn-based DAMON sysfs selftest to reduce
duplicated and easy-to-mistake dumping-and-checking pattern.

Final two patches fix up sphinx and grammartical errors.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus_npicx3q7 | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus_npicx3q7

diff --git a/hkml_cv_bogus/hkml_cv_bogus_npicx3q7 b/hkml_cv_bogus/hkml_cv_bogus_npicx3q7
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.47.3

